### PR TITLE
Add detailed information in top-level README

### PR DESCRIPTION
**Pull request by razvand**
Document repository structure. Add information on using the content and contributing to it. Reference support channels (GitHub Discussions, Discord).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# SSS TODO_track_name
+
+This is the public repository for the **TODO** track of the Security Summer School (SSS).
+It contains information available to participants: session content, slides, activities support files.
+Apart from SSS participants, you can use this content by yourself to learn more about **TODO**.
```
Comment (carrrina) [inline README.md:5]: What are you supposed to replace this **TODO** with (web security, binary exploiting...)? Maybe a list of options in brackets would help the one who uses this template, to me it was unclear.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+## Using the Content
+
+As with all SSS tracks, the **TODO** track consists of multiple sessions.
+One session typically covers a particular topic of the track.
+Ideally you would go through sessions orderly, but you can jump ahead to a topic of interest.
```
Comment (carrrina) [inline README.md:11]:
```suggestion
Ideally you would go through the sessions orderly, but you can jump ahead to a topic of interest.
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+Each session has its own folder.
+Access the `README.md` file in each session folder.
```
Comment (carrrina) [inline README.md:14]:
```suggestion
The `README.md` file from a session folder details the topic of the session and includes practical demos of tools and snippets of code.
```
(Or something like that - I didn't like the short, repetitive, separate sentences.)

File: README.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+It details the topic of the session and includes practical demos of tools and snippets of code.
```
Comment (carrrina) [inline README.md:15]:
```suggestion
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+  the rest is up to you.
+  Even so, challenge activity folders store the reference solution for you to take a sneak peak in case you're lost.
+
+## Support
+
+In case you have questions or want to know more about a topic, use the the `Discussions` tab or join the [SSS Discord community](https://discord.gg/qx3RmNEVkB).
+
+## Sessions
+
+The session for the SSS **TODO** track are:
```
Comment (carrrina) [inline README.md:35]:
```suggestion
The sessions for the SSS **TODO** track are as follows:
```

File: README.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+An activity folder consists of:
+
+* `public/`: support file for the activity: binary files, helper scripts, packet captures etc.
+* `sol`/: the reference solution of the activity;
+  Tutorial activities feature a `README.md` file in the solution folder, with a detailing of the solution.
+
+## Contributing
+
+Contributions are welcome.
+See [the contribution guide](CONTRIBUTING.md) on how you could report or fix issues and on how you can improve content.
```
Comment (carrrina) [inline README.md:61]:
```suggestion
See the [contribution guide](CONTRIBUTING.md) on how you could report or fix issues and on how you can improve the content.
```

File: README.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+Reviewers are requested to follow [the reviewing guide](REVIEWING.md).
```
Comment (carrrina) [inline README.md:63]:
```suggestion
Reviewers are requested to follow the [reviewing guide](REVIEWING.md).
```

File: README.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+* `activities/`: each activity has its own folder with support files (including the reference solution)
+* `assets/`: support files for session contents (images, diagrams, code snippets, packet captures etc.)
+* `slides/`: session slides
+
+### Activity Folder Structure
+
+An activity folder consists of:
+
+* `public/`: support file for the activity: binary files, helper scripts, packet captures etc.
+* `sol`/: the reference solution of the activity;
```
Comment (carrrina) [inline README.md:55]:
```suggestion
* `sol`/: the reference solution of the activity
```
In the other lists you didn't use a `;`

File: README.md [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+
+After reading the session contents, jump to the hands-on activities.
+A typical activity will challenge you to solve / exploit a given issue and obtain a flag.
+Read the activity description in the session contents and then use the activity support file to get to a solution.
+Each activity will develop your practical skills and help you get a better understanding of the session topics.
+
+Activities come in two forms:
+
+* **tutorials** that provide a step-by-step detailing of the solution
+* **challenges** that only state the goal and provide an initial skeleton (and maybe some hints);
```
Comment (carrrina) [inline README.md:25]:
```suggestion
* **challenges** that only state the goal and provide an initial skeleton (and maybe some hints), then the rest is up to you
```
I didn't like the unconsistent use of `;` and `.`

File: README.md [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+  the rest is up to you.
```
Comment (carrrina) [inline README.md:26]:
```suggestion
```

**Review by carrrina — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (razvand) [reply README.md:5]: Gotcha, I will clear that.

**Review by carrrina — APPROVED** [review]

**Pull request merged**